packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -215,6 +215,8 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
     return (
       <Tag
         role="presentation"
+        aria-labelledby={ariaLabelledBy}
+        aria-describedby={ariaDescribedBy}
```

Comment:
I think the existing design here is problematic for a couple of reasons:
* FocusZone defines `ariaLabelledBy` and `ariaDescribedBy` but does no special processing on them. Components should inherently be supporting `aria-` attributes and we shouldn't have to explicitly add them to props. 
* The old implementation (the original lines removed below) actually overwrites any values of `aria-labelledby` or `aria-describedby` by the user, even when `ariaDescribedBy` and `ariaLabelledBy` are undefined, which doesn't seem good.

I think we should deprecate these props as part of this PR and direct people to use the native attributes. The reordering in this PR is good because it ensures that native versions of these props will also get priority.

Reply:
Totally agree, deprecated both `ariaLabelledBy` and `ariaDescribedBy`.